### DISCUSSION TITLE
Backend

### DISCUSSION THREAD

**Discussion by AnkitaMalik22** [AnkitaMalik22/devmate#40] [Q&A]
If anyone want to work on the backend part feel free to share here .

** Next plans **

-  Register & Login page 
-  user authentication
-  Add profiles to favourites(a new page)

**Comment by Harry-Ramsey** [comment]
Hello, this seems like an interesting project to contribute to. I wanted to open up a discussion on what database should be used for the backend? Obviously, this is quite important as we will need to store information about users and depending upon the number of users, the database will need to scale too, mostly with read requests.

**Reply by AnkitaMalik22** [reply]
I'm quite familiar with Mongodb & MySQL!
What you think?

**Reply by Harry-Ramsey** [reply]
I think MongoDB could work quite well here as user profiles are already stored in JSON format and we could keep this format using Mongo document model.  I will wait a bit in case anyone else wants to put forward a good point about the two or maybe another one.

**Reply by AnkitaMalik22** [reply]
I agree with You ! Mongodb will be good to use in this case

**Reply by sahilarora02** [reply]
Yes, definitely Mongodb should be use

**Reply by Harry-Ramsey** [reply]
Could we have an issue created for MongoDB? It's quite important and we should attempt to get the design right first before fully implementing it.

**Reply by AnkitaMalik22** [reply]
> Could we have an issue created for MongoDB? It's quite important and we should attempt to get the design right first before fully implementing it.

Yes
